src/jdk.jdwp.agent/unix/native/libjdwp/linker_md.c

```diff
@@ -90,13 +98,13 @@ dbgsysBuildLibName(char *holder, int holderlen, const char *pname,
     const int pnamelen = pname ? strlen(pname) : 0;
 
     *holder = '\0';
```

Comment:
This assignment is now redundant: please remove it.

Reply:
Will do.

Reply:
Done.